setup.py

```diff
@@ -17,7 +17,8 @@ def read(fname):
     'pyramid_basemodel',
     'tzf.pyramid_yml >=0.2',
     'pyramid >=1.5a1',
-    'pyramid_mako'
+    'pyramid_mako',
+    'pycountry==1.20'
```

Comment:
Please define exact version in requirements-tests.txt files. 

the requirements-tests.txt files is used to test with every new pycountry release, while placing it in setup py will cause any developer using this package to rely on pyramid_localize upgrade to be able to upgrade pycountry.